wallaby.js

```diff
@@ -0,0 +1,32 @@
+process.env.NODE_ENV = 'test';
+
+var wallabyWebpack = require('wallaby-webpack');
+var webpackConfig = require('./webpack.config');
+webpackConfig.externals = {
+    'react': 'React'
+};
+var wallabyPostprocessor = wallabyWebpack(webpackConfig);
+
+module.exports = function(wallaby) {
+    return {
+        files: [
+            {pattern: 'node_modules/chai/chai.js', instrument: false},
+            {pattern: 'node_modules/phantomjs-polyfill/bind-polyfill.js', instrument: false},
+            {pattern: 'node_modules/react/dist/react-with-addons.js', instrument: false},
+            {pattern: 'src/**/*.js', load: false},
+            {pattern: 'src/**/*.spec.js', ignore: true},
+            {pattern: 'src/client/**/*.js', ignore: true}
+        ],
+        tests: [
+            {pattern: 'src/**/*.spec.js', load: false}
+        ],
+        compilers: {
+            '**/*.js': wallaby.compilers.babel(),
```

Comment:
Do we need to pass in our babel config since its not in the `.babelrc` file and instead in `package.json`?